react-backend/client/src/shared/components/Dots.js

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+
+export default styled.div`
+  z-index: -1;
+  left: 50%;
+  top: 0;
+  transform: translate3d(-50%, 0, 0);
+
+  &, &::before, &::after {
```

Comment:
What does this do?

Reply:
I don't know, I just copied it from their website 🤷‍♂️

Reply:
Haha coding in a nutshell